src/lib.rs

```diff
@@ -540,3 +542,42 @@ impl Default for WindowAttributes {
         }
     }
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct VideoMode {
+    dimensions: (u32, u32),
+    bit_depth: u16,
```

Comment:
Is it possible to expose the number of bits for each channel separately? At least distinguish the number of color bits from alpha bits?

Reply:
WinAPI does not expose this information as far as I can tell. SDL has some heuristics for this, but I think those were written with 16-bit and 8-bit modes in mind for older Windows systems. There's still some code for greyscale support in there.. Windows 8 and up no longer supports these modes. What do you need this information for?

Reply:
No, reason, I just noticed glfw exposed the separate bits. That's OK.